src/Umbraco.Web.Common/RuntimeMinification/SmidgeComposer.cs

```diff
@@ -11,6 +16,24 @@ public sealed class SmidgeComposer : IComposer
         public void Compose(Composition composition)
         {
 
+            /////////////////////////////////////////
+            // Begin insanity
+            /////////////////////////////////////////
+
+            // TODO: MSDI - no idea what's up with smidge
+            // Obviously this is the stupidest stuff anyone has ever written, HOWEVER, it enables me to render that install page.
+
+            var register = typeof(Composition).GetField("_register", BindingFlags.NonPublic | BindingFlags.Instance).GetValue(composition);
```

Comment:
Yeah, once a pattern is established with the composition API that passes the service collection this. should be easy.

Reply:
It should be easy now too, that composer seems to work fine with LightInject but with MSD a failure occurs to resolve a BundleManager when resolving a SmidgeRuntimeMinifier.

I do hope this commit gets removed and replaced with something a little less dumb, just wanted to go to bed.

Reply:
There are two extra smidge related services registered in MSDI vs lightinject

+ Umbraco.Web.Common.RuntimeMinification.SmidgeNuglifyJs + Umbraco.Web.Common.RuntimeMinification.SmidgeHelperAccessor